site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -199,24 +203,25 @@ async function openTabOnRight() {
   // When not wrapped in try/catch, errors thrown in an async
   // function will propagate down the promise chain
   let tabs = await chrome.tabs.query(queryOptions);
-
   if (!tabs.length) return;
+
   let tab = await chrome.tabs.create({
     url: 'https://example.com',
     index: tab[0].index + 1,
   });
-
   if (!tab) return;
+
   console.log('tab created', tab);
   return tab;
 }
 ```
 
 {% Aside %}
+
 Note that `await` is only valid in async functions and the top-level bodies of modules.
```

Comment:
Could you add something like:

You can ensure your background Service Worker is run as a module by setting `"type": "module"` in the `"background"` key of your manifest.

With "manifest" linking to https://deploy-preview-1671--developer-chrome-com.netlify.app/docs/extensions/mv3/manifest/.
(That page hilariously has had `"type":` dropped from the `"module"` bit)?

Reply:
Applied your suggestion with some slight tweaks. Also fixed the manifest issues you mentioned.